lib/cloud_controller/config_schemas/base/api_schema.rb

```diff
@@ -333,6 +333,7 @@ class ApiSchema < VCAP::Config
             max_annotations_per_resource: Integer,
 
             internal_route_vip_range: String,
+            cpu_weight_scalar: Float,
```

Comment:
this shouldn't be in the schema, as it is not required (or exposed) for Diego foundations